utils/io/Makefile

```diff
@@ -6,7 +6,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=io
-PKG_RELEASE:=2
+PKG_RELEASE:=4
```

Comment:
This should be 3

Reply:
I increased it for each patch, but I can also only increase it in the last one.

Reply:
It makes sense to do it once. A version in between will never be released.